examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -0,0 +1,634 @@
+    "\n",
+    "The first step is to let the jupyter notebook connect to cluster. To achieve that, import the ipypyarallel library since we rely on it to manager the cluster, then connect it through `ipp.Client(profile=\"bluefog\")` as shown in the following cell.\n",
+    "\n",
+    "`rc` (remote client) is the key that you will use to send the code from this notebook (driver) to the cluster namely multiple workers. \n",
+    "To check the notebook has connected to the worker succesfully, you can look the ID of remote clients through `rc.ids`\n",
+    "Depending on how many processes you starts, the length of rc will be the same number as the number of process you start the `ibfrun`, i.e. the number after `-np`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-01-02T01:57:45.309973Z",
+     "start_time": "2021-01-02T01:57:45.285024Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[0, 1, 2, 3]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ipyparallel as ipp\n",
+    "rc = ipp.Client(profile=\"bluefog\")\n",
+    "rc.ids"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you saw the `ids` is an empty list, this means ipyparallel cannot connect it with the clusters. You can stop the `ibfrun` by `ctrl+c`. Kill all remainning `ipcontroller` or `mpi` processes through `ps` if necessary. Then, try to rerun the `ibfrun` again."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Parallel execution through `px`\n",
+    "\n",
+    "`px` -- parallel execution is a [magic command](https://ipython.readthedocs.io/en/stable/interactive/magics.html) in IPython environment similar as the `%matplotlib` or `%%timeit`. If you are not familar with magic command, you can think it as a special command in IPython that will change the behavior of IPython to interprete the following python code. As the same for all magic commands, `%px` is for one line and `%%px` is for whole cell. For example \n",
+    "`%px print(\"Hello world\")` will put the `print(\"Hello world\")` to run under the `px` environment. And, if you put the `%%px` at the top of one cell, the rest code in that cell all will be run under the `px` environment.\n",
+    "\n",
+    "The code under the `px` env will be executed into all each worker in parallel instead of running in this notebook.\n",
+    "It is important to rememebr that each worker are one seperated IPython environment as well. The following is hello-world example to work with bluefog in jupyter notebook, which is already sufficient for most  usage cases. You can find more advanced usage of `px` in the [document page](https://ipyparallel.readthedocs.io/en/latest/magics.html).\n"
```

Comment:
- each worker are one -> each worker is one
- hello-world example -> a hello-world example